lib/block-supports/layout.php

```diff
@@ -54,14 +55,34 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 		$wide_max_width_value = wp_strip_all_tags( explode( ';', $wide_max_width_value )[0] );
 
 		if ( $content_size || $wide_size ) {
-			$style  = "$selector > :where(:not(.alignleft):not(.alignright)) {";
+			$style  = "$selector > :where(:not(.alignleft):not(.alignright):not(.alignfull)) {";
 			$style .= 'max-width: ' . esc_html( $all_max_width_value ) . ';';
 			$style .= 'margin-left: auto !important;';
 			$style .= 'margin-right: auto !important;';
 			$style .= '}';
 
 			$style .= "$selector > .alignwide { max-width: " . esc_html( $wide_max_width_value ) . ';}';
 			$style .= "$selector .alignfull { max-width: none; }";
+
+			if ( isset( $block_spacing ) ) {
+				$block_spacing_values = gutenberg_style_engine_get_block_supports_styles(
+					array(
+						'spacing' => $block_spacing,
+
```

Comment:
Tiniest of tiny nits, but is this an extra unneeded line on whitespace?

Reply:
It was!